packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -52,6 +53,7 @@ enum _ScaffoldSlot {
   snackBar,
   floatingActionButton,
   drawer,
+  statusBar,
```

Comment:
On iOS you need to tap in the status bar to scroll. Tapping on the rest of the app bar doesn't scroll.